hacking/shippable/download.py

```diff
@@ -62,28 +76,12 @@ def parse_args():
                         action='store_true',
                         help='show what would be downloaded without downloading')
 
-    parser.add_argument('--key',
-                        dest='api_key',
-                        default=api_key,
-                        required=api_key is None,
-                        help='api key for accessing Shippable')
-
-    parser.add_argument('--console-logs',
```

Comment:
Can you restore this functionality to the script? Although it's not used for incidental coverage, it does get used for other tasks.

Reply:
Sure, on it